code/controllers/subsystem/pai.dm

```diff
@@ -3,196 +3,100 @@ SUBSYSTEM_DEF(pai)
 
 	flags = SS_NO_INIT|SS_NO_FIRE
 
+	var/datum/pai_candidate/candidate
```

Comment:
This doesn't make sense, this is a subsystem, why is there only one?

Reply:
The rest of the code surrounding this relating to UI on the SS is...weird!

I would recommend the more common pattern of just creating a `/datum/pai_menu` or whatever per mob, and using that.

Reply:
Good point. I think this menu exists with ```recruitWindow```, but I was having trouble because candidate was a ref. Since src is working, I won't need this.